sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Features Added
 
 - With the dropping of support for Node.js versions that are no longer in LTS, the dependency on `@types/node` has been updated to version 12. Read our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+- Updated our internal core package dependencies to their latest versions in order to add support for Opentelemetry 1.0.0 which is compatible with the latest versions of our other client libraries.
```

Comment:
nit: OpenTelemetry uses a capital `T`.

Also I wonder if "which is compatible with the latest versions of our other client libraries." is needed? It makes the sentence kind of confusing to me.

```suggestion
- Updated our internal core package dependencies to their latest versions in order to add support for OpenTelemetry 1.0.0 which is compatible with the latest versions of our other client libraries.
```

Reply:
We used this text in other packages already. What we want to convey is that the latest versions of all our client libraries are now compatible with Open Telemetry 1.0.0

cc @maorleger for more on this